smart-contracts/contracts/Unlock.sol

```diff
@@ -259,25 +259,6 @@ contract Unlock is
     return lock.isLockManager(_sender);
   }
 
-  /**
-   * This function returns the discount available for a user, when purchasing a
-   * a key from a lock.
-   * This does not modify the state. It returns both the discount and the number of tokens
-   * consumed to grant that discount.
-   * TODO: actually implement this.
-   */
-  function computeAvailableDiscountFor(
```

Comment:
I think we need to keep it, but just not call it anymore from the latest version of Unlock, otherwise "old" locks will break :/

Reply:
Let's make sure we add a comment to show it as deprecated but kept for backward compat